CalibTracker/SiStripCommon/plugins/ShallowEventDataProducer.cc

```diff
@@ -22,7 +23,7 @@ produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
 
 
   edm::Handle< L1GlobalTriggerReadoutRecord > gtRecord;
-  iEvent.getByLabel( edm::InputTag("gtDigis"), gtRecord);
+  iEvent.getByLabel( edm::InputTag(trig_label_), gtRecord);
```

Comment:
don't make an InputTag every event - instead, make it once, or better yt, use the output of the consumes call and use getByToken(consumesOutput,gtRecord)
Thanks!
@mverzett